hydromt_wflow/naming.py

```diff
@@ -342,6 +342,10 @@
         "wflow_v1": "floodplain_water__sum_of_volume-per-depth",
         "hydromt_name": "floodplain_volume",
     },
+    "floodplain_roughness": {
```

Comment:
Harmonise default name with river equivalent (river_manning_n)
```suggestion
    "floodplain_manning_n": {
```